metadata.platform.in

```diff
@@ -23,6 +23,12 @@ version=@@FDO_RUNTIME_VERSION@@
 directory=lib/extensions/gstreamer-1.0
 subdirectories=true
 
+[Extension org.freedesktop.Platform.Icontheme]
+version=1.0
+directory=share/runtime/share/icons
+subdirectories=true
```

Comment:
you probably also want `autodelete=true`, otherwise the extension won't get automatically removed.